### PR TITLE
Add ccache Github workflow 

### PR DESCRIPTION
**Pull request by inknos**

### CODE REVIEW

File: .github/workflows/ccache-build.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      # this step needs to be incorporated in our container that we use for testing
+      - name: Install dependencies
+        run: |
+          dnf install -y ccache clang && dnf builddep -y ./dnf5.spec
```
Comment (inknos) [inline .github/workflows/ccache-build.yaml:26]: @m-blaha about this, would you rather update our current container that we use for CI or should we create a separate image for building?

**Comment by inknos** [comment]
looks like the names the workflows are not updated. I'll retry to run them tomorrow

File: .github/workflows/ccache-build.yaml (hunk 1 above)
Comment (m-blaha) [reply .github/workflows/ccache-build.yaml]: I'd prefer to keep one container.

**Review by m-blaha — APPROVED** [review]
LGTM, thanks!

File: .github/workflows/ccache-build.yaml (hunk 1 above)
Comment (inknos) [reply .github/workflows/ccache-build.yaml]: also, the COPR secret needs to be fixed. I'll do those two fixes before merging

**Pull request merged**